runtime/gc_base/modronapi.cpp

```diff
@@ -1131,6 +1131,9 @@ j9gc_reinitialize_for_restore(J9VMThread *vmThread, const char **nlsMsgFormat)
 	J9JavaVM *vm = vmThread->javaVM;
 	J9MemoryManagerVerboseInterface *mmFuncTable = (J9MemoryManagerVerboseInterface *)vm->memoryManagerFunctions->getVerboseGCFunctionTable(vm);
 
+	Assert_MM_true(NULL != extensions->getGlobalCollector());
+	Assert_MM_true(NULL != extensions->configuration);
+
 	PORT_ACCESS_FROM_JAVAVM(vm);
 
 	if (!j9gc_reinitializeDefaults(vmThread)) {
```

Comment:
notice this is an inter module method, but it does not seem like it has to be
also should it have 'for restore' in the name ?
btw original method is gcInitializeDefaults (no j9 in the name)

Reply:
>noticed this is an inter module method, but it does not seem like it has to be

Removed from `J9MemoryManagerFunctions`

>also should it have 'for restore' in the name ?
btw original method is gcInitializeDefaults (no j9 in the name)

Changed `j9gc_reinitializeDefaults` -> `gcReinitializeDefaultsForRestore`